packages/runtime/src/helpers/flags.ts

```diff
@@ -11,7 +13,19 @@ import destr from 'destr'
  * If disabled, we bundle all API Routes into a single function.
  * This is can lead to large bundle sizes.
  *
+ * Relies on `next-server.js.nft.json`, which is only supported in Next.js 12+.
+ *
  * Disabled by default. Can be overriden using the NEXT_SPLIT_API_ROUTES env var.
  */
-export const splitApiRoutes = (featureFlags: Record<string, unknown>): boolean =>
-  destr(process.env.NEXT_SPLIT_API_ROUTES) ?? featureFlags.next_split_api_routes ?? false
+export const splitApiRoutes = (featureFlags: Record<string, unknown>, publish: string): boolean => {
```

Comment:
This looks good. Can you add some tests for this?

Reply:
done in https://github.com/netlify/next-runtime/pull/2125/commits/efa9787fd2c4adae4df8341ee545b4abb7596f9f.